### DISCUSSION TITLE
custom facet sort function

### DISCUSSION THREAD

**Discussion by pdebruic** [observablehq/plot#1346] [Q&A] [answered]
I have categorical names that don't make as much sense alphabetically.  Would be nice to give facet a sort function instead of the default.

**Comment by mbostock** [comment, marked as answer]
You can specify the domain of the *fx* or *fy* scale explicitly to change the order. And you can use the mark **sort** option to impute the domain of these scales from data. See the barley plot example here: https://observablehq.com/@observablehq/plot-facets
